src/autotester/server/utils/tests/path_management_test.py

```diff
@@ -0,0 +1,23 @@
+from src.autotester.server.utils.path_management import current_directory, add_path
+import tempfile
+import os
+import sys
+
+
+def test_current_directory():
+    tmp_dir = tempfile.gettempdir()
+    curr_dir = os.getcwd()
+    with current_directory(tmp_dir):
+        temp_dir = os.getcwd()
```

Comment:
`tmp_dir` and `temp_dir` are very similar. Please use variable names that are distinct and descriptive